webapp/channels/src/components/system_notice/__snapshots__/system_notice.test.tsx.snap

```diff
@@ -1,342 +1,73 @@
-<div
-  className="system-notice bg--white"
->
-  <div
-    className="system-notice__logo"
-  >
-    <MattermostLogo />
-  </div>
-  <div
-    className="system-notice__body"
-  >
-    <div
-      className="system-notice__title"
-    >
-      some title
-    </div>
-    some body
-    <div
-      className="system-notice__footer"
-    >
-      <button
-        className="btn btn-sm btn-primary"
-        id="systemnotice_remindme"
-        onClick={[Function]}
-      >
-        <MemoizedFormattedMessage
-          defaultMessage="Remind Me Later"
-          id="system_notice.remind_me"
-        />
-      </button>
-      <button
-        className="btn btn-sm btn-tertiary"
-        id="systemnotice_dontshow"
-        onClick={[Function]}
-      >
-        <MemoizedFormattedMessage
-          defaultMessage="Don't Show Again"
-          id="system_notice.dont_show"
-        />
-      </button>
-    </div>
-  </div>
-</div>
+<ContextConsumer>
+  <Component />
+</ContextConsumer>
 `;
 
 exports[`components/SystemNotice should match snapshot for with allowForget equal false 1`] = `
-<div
-  className="system-notice bg--white"
->
-  <div
-    className="system-notice__logo"
-  >
-    <MattermostLogo />
-  </div>
-  <div
-    className="system-notice__body"
-  >
-    <div
-      className="system-notice__title"
-    >
-      some title
-    </div>
-    some body
-    <div
-      className="system-notice__footer"
-    >
-      <button
-        className="btn btn-sm btn-primary"
-        id="systemnotice_remindme"
-        onClick={[Function]}
-      >
-        <MemoizedFormattedMessage
-          defaultMessage="Remind Me Later"
-          id="system_notice.remind_me"
-        />
-      </button>
-    </div>
-  </div>
-</div>
+<ContextConsumer>
+  <Component />
+</ContextConsumer>
 `;
 
 exports[`components/SystemNotice should match snapshot when a custom icon is passed 1`] = `
-<div
-  className="system-notice bg--white"
->
-  <div
-    className="system-notice__logo"
-  >
-    <span>
-      icon
-    </span>
-  </div>
-  <div
-    className="system-notice__body"
-  >
-    <div
-      className="system-notice__title"
-    >
-      some title
-    </div>
-    some body
-    <div
-      className="system-notice__footer"
-    >
-      <button
-        className="btn btn-sm btn-primary"
-        id="systemnotice_remindme"
-        onClick={[Function]}
-      >
-        <MemoizedFormattedMessage
-          defaultMessage="Remind Me Later"
-          id="system_notice.remind_me"
-        />
-      </button>
-      <button
-        className="btn btn-sm btn-tertiary"
-        id="systemnotice_dontshow"
-        onClick={[Function]}
-      >
-        <MemoizedFormattedMessage
-          defaultMessage="Don't Show Again"
-          id="system_notice.dont_show"
-        />
-      </button>
-    </div>
-  </div>
-</div>
+<ContextConsumer>
+  <Component />
+</ContextConsumer>
```

Comment:
can you please try to mount the component instead of shallow rendering, so we can get a bit more deeper jest snapshots

Reply:
Done